### PR TITLE
Update label API

### PR DESCRIPTION
**Pull request by ScorpioCPH**
- do not check label name
- fix duplicate error check
- remove labels of this project while delete project

### CODE REVIEW

**Comment by ScorpioCPH** [comment]
@zxydaisy, please review my code, thanks!

File: Deploy/db/registry.sql [hunk 1]
```diff
@@ -62,8 +62,8 @@ create table project (
  project_id int NOT NULL AUTO_INCREMENT,
  owner_id int NOT NULL,
  name varchar (30) NOT NULL,
- manager varchar (50) NOT NULL,
- remark varchar (500) NOT NULL,
+ manager varchar (50),
+ remark varchar (500),
```
Comment (zxydaisy) [inline Deploy/db/registry.sql]: remark还是改成必填的吧

**Comment by zxydaisy** [comment]
@ScorpioCPH  review完了

File: Deploy/db/registry.sql (hunk 1 above)
Comment (ScorpioCPH) [inline Deploy/db/registry.sql]: 恩，这里当成中文名称好了

**Comment by ScorpioCPH** [comment]
fix https://github.com/caicloud/harbor/issues/12

**Pull request merged**